### PR TITLE
[AIRFLOW-6938] Don't read dag_directory in SchedulerJob

### PR DESCRIPTION
**Pull request by mik-laj**
This change separates two SchedulerJob from files. I want to sort out the code.

---
Issue link: [AIRFLOW-6938](https://issues.apache.org/jira/browse/AIRFLOW-6938)

Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by ashb — APPROVED** [review]

File: airflow/utils/dag_processing.py [hunk 1]
```diff
@@ -577,7 +566,7 @@ def __init__(self,
 
         self._last_zombie_query_time = None
         # Last time that the DAG dir was traversed to look for files
-        self.last_dag_dir_refresh_time = timezone.utcnow()
+        self.last_dag_dir_refresh_time = timezone.make_aware(datetime.fromtimestamp(0))
```
Comment (kaxil) [inline airflow/utils/dag_processing.py:569]: Why this change?

Comment (mik-laj) [reply airflow/utils/dag_processing.py:569]: We want the files to be read during the first loop cycle.
https://github.com/apache/airflow/blob/master/airflow/utils/dag_processing.py#L717-L718

**Review by kaxil — APPROVED** [review]
